hive-services/hive.yaml

```diff
@@ -1,5 +1,5 @@
 services:
-- hash: 07d084afb85b009d02c421f2e1c11e7ae27bb622
+- hash: 2342f033ce2b8f695ac3997856f938a03005dec8
```

Comment:
Is this the hash of latest container image of `hive` in registry?

Reply:
@LalatenduMohanty in the app-sre pipelines, when we build the image, we always tag it with the hash shortened to [7](https://github.com/app-sre/saas-hive/blob/master/hive-services/hive.yaml#L6) chars.

- Hive commit: https://github.com/openshift/hive/tree/2342f033ce2b8f695ac3997856f938a03005dec8
- Quay tags: https://quay.io/repository/app-sre/hive?tab=tags